osler/workup/models.py

```diff
@@ -200,10 +139,7 @@ class Meta:
         related_name="other_volunteer",
         help_text="Which other volunteer(s) did you work with (if any)?")
 
-    clinic_day = models.ForeignKey(
-        ClinicDate,
-        on_delete=models.PROTECT,
-        help_text="When was the patient seen?")
+    encounter = models.ForeignKey(Encounter, on_delete=models.CASCADE)
```

Comment:
I was looking at how the other models using encounter store their own date, and I feel like it could be useful for to also store written_datetime separately for workup as well. It seems like there could be instances when it is medically important to know exactly when a patient was seen. It also feels more consistent

Reply:
you are very right